configs/defaults/rd_combiner.toml

```diff
@@ -28,20 +28,15 @@ indel_recal_disc_size = 20
 snps_gather_disc_size = 10
 snps_recal_disc_size = 20
 
+# without this setting we use the native Hail version in the container
+# not currently in use as the cpg-utils default contains these patches by default, plus extra logging
+# most recently set to 'a8be268326b2ac168035b629a78465c9d94fc7b9' - 0.2.133 + 400'retry + StreamConstraints patch
+#default_jar_spec_revision = 'a8be268326b2ac168035b629a78465c9d94fc7b9'
```

Comment:
Please don't check references to `workflow.default_jar_spec_revision` into production-pipelines config files. We want to be able to update that in cpg-infrastructure as necessary, and it will lead to confusion and unsupportable configurations if various workflows override it.

Reply:
On the basis that this is a commented out section of the config, with an explanatory comment saying "if you need to change the JAR spec, this is how you'd do it", I'm going to leave it in...

We currently have an undocumented, optional, and occasionally essential behaviour in cpg-utils. Surely more information is better than less.